index.html

```diff
@@ -1432,9 +1432,21 @@ <h4>TTML Format</h4>
       <p class="issue" data-number="25"></p>
       <section>
         <h4>Proprietary Metadata</h4>
-        <p>Many dubbing and <a>audio description</a> workflows permit annotation of <a>Script Events</a> or documents with proprietary metadata.
-          Additional vocabulary MAY be included if it is in namespaces not defined by this specification, [[TTML2]] or [[ttml-imsc1.2]].</p>
-        <p class="issue" data-number="31"></p> 
+        <p>Many dubbing and <a>audio description</a> workflows permit annotation of <a>Script Events</a> or documents with proprietary metadata. Exsiting metadata vocabulary from [[TTML2]] MAY be included. Additional vocabulary in proprietary namespaces MAY also be included.</p>
+        <p class="note">It is possible to add information such as the title of the programme using [[TTML2]] constructs.
+        <pre class="example">
+...
+&lt;medadata&gt;
```

Comment:
I thought you'd accepted the fix on the typo here but it seems to have come back!